lib/packExternalModules.js

```diff
@@ -83,14 +83,14 @@ function getProdModules(externalModules, packagePath, dependencyGraph, forceExcl
     if (moduleVersion) {
       prodModules.push(`${module.external}@${moduleVersion}`);
 
+      let nodeModulesBase = path.dirname(path.join(process.cwd(), packagePath));
+
+      if (nodeModulesRelativeDir) nodeModulesBase = path.join(process.cwd(), nodeModulesRelativeDir);
```

Comment:
I prefer that kind of way for readability
```suggestion
      if (nodeModulesRelativeDir) {
        nodeModulesBase = path.join(process.cwd(), nodeModulesRelativeDir);
      }
```

Reply:
Thanks @j0k3r 
comments have been addressed.